client/js/navigation/RootStackNavigator.js

```diff
@@ -18,7 +21,10 @@ const AppStack = createStackNavigator(
     DrawerNavigationLayout,
     Beer: BeerModal,
     Event: EventModal,
-    StampsReceived: StampsReceivedModal
+    StampsReceived: StampsReceivedModal,
+    RedeemInfo: RedeemInfoModal,
+    RedeemBarcode: RedeemBarcode,
+    RedeemSuccess: RedeemSuccess
```

Comment:
If redeem barcode and redeem success are modal, please add a postfix in their name. Thank you.